src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinder.cs

```diff
@@ -135,7 +148,13 @@ protected virtual bool CanBindProperty(ModelBindingContext bindingContext, Model
         /// </returns>
         protected virtual Task BindProperty(ModelBindingContext bindingContext)
         {
-            return bindingContext.OperationBindingContext.ModelBinder.BindModelAsync(bindingContext);
+            IModelBinder binder;
+            if (_propertyBinders.TryGetValue(bindingContext.ModelMetadata, out binder))
```

Comment:
The provider won't leave any property out. So why isn't it an error for `TryGetValue()` to fail?